### ISSUE TITLE
Popup not working

### ISSUE THREAD

**Issue by DEfusion** [Semantic-Org/Semantic-UI-React#1255]
### Steps
I copied the first example from the documentation:

```
<Popup
    trigger={<Button icon='add' />}
    content='Add users to your feed'
  />
```

When I hovered over the `Button` I get this error:

`Uncaught TypeError: Cannot read property 'appendChild' of null
    at Portal._this.mountPortal (eval at <anonymous> (vendor-bundle.js:4693), <anonymous>:244:18)`

Doing some digging I found that it wasn't picking up the mount node:

```
Portal.defaultProps
=> Object {closeOnDocumentClick: true, closeOnEscape: true, openOnTriggerClick: true, mountNode: null}
```

I looked at the code and the `isBrowser` code does return true for me in browser (though I copied and pasted it into my react component to test this).

In order to get the popup to work I have to set the prop `mountNode={document.body}`

### Version
v0.18.1

I couldn't get this to fail for me in a codepen though so I'm not sure what's going on.

**Comment by levithomason** [comment]
I can't imagine `v0.18.1` is the actual version of SUIR you're using :)  Could you confirm?

**Comment by DEfusion** [comment]
Sorry don't know where I got that number from `v0.64.4` is what I'm on :)

**Comment by SirWaddles** [comment]
I'm having the same issue when using a Dimmer with the page prop.

Running `0.64.4`

**Comment by SirWaddles** [comment]
As far as I can tell, it looks like [this commit](https://github.com/Semantic-Org/Semantic-UI-React/pull/806/commits/c931a1fd1aaa23a5cf004faae12c3bf5ac254186) is to blame.

In `Portal.js`, `defaultProps` is a static member of the class. These are evaluated on first execution.

the statement `mountNode: isBrowser ? document.body : null` is evaluated before the page loads, so `document.body` is null at that time.

**Comment by levithomason** [comment]
Good eye, this should move to an inline `||`operator instead.

**Comment by SirWaddles** [comment]
My Pull Request only fixes the Dimmer, and probably not in the best way. It just seemed to solve my problem.

My problem is also potentially that I have the `<script>` tag in the `<head>` element as opposed to at the bottom of the page.

**Comment by levithomason** [comment]
This should be fixed by #1267, thanks @SirWaddles.
